language/stdlib/transaction_scripts/rotate_validator_config.move

```diff
@@ -1,5 +1,4 @@
 script {
-    use 0x1::LibraSystem;
```

Comment:
Wondering: do we need both this and `rotate_consensus_pubkey`?

Reply:
You are right, we don't need the rotate_consensus_pubkey. Nunced it.